eating_cookies/eating_cookies.py

```diff
@@ -6,7 +6,23 @@
 # a solution that is more efficient than the naive 
 # recursive solution
 def eating_cookies(n, cache=None):
```

Comment:
Great job on this solution, @nia-ja ! 👍 
<img width="1920" alt="Screen Shot 2020-03-05 at 8 21 46 PM" src="https://user-images.githubusercontent.com/16110931/76041033-1b9b2a00-5f1f-11ea-9851-0092b2ce97f2.png">
How could you optimize your solution so that when you run the `python test_eating_cookies.py -k large` command in the terminal, the tests for the large input won't take forever?